langwatch_nlp/tests/studio/test_parse.py

```
@@ -306,3 +306,139 @@ def test_parse_component_raises_for_end_node(self):
+import dspy
+
+class Helper:
+    def __call__(self, x):
+        return x
+
+class Code(dspy.Module):
+    def forward(self, **kwargs):
+        return {"output": "main"}
+"""
+        )
+        code, class_name, _ = parse_component(node, basic_workflow)
```

Ruff (0.15.12)

[warning] Unpacked variable `code` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)